openid-4-verifiable-presentations-1_0.md

```diff
@@ -2348,19 +2181,31 @@ The security properties that are normally provided by the Client Identifier are
 
 The audience for the response (for example, the `aud` value in a Key Binding JWT) MUST be the Origin, prefixed with `origin:`, for example `origin:https://verifier.example.com/`. This is the case even for signed requests. Therefore, when using OpenID4VP over the DC API, the Client Identifier is not used as the audience for the response.
 
-# Credential Format Specific Parameters {#format_specific_parameters}
+# Credential Format Specific Parameters and Rules {#format_specific_parameters}
 
-OpenID for Verifiable Presentations is Credential Format agnostic, i.e., it is designed to allow applications to request and receive Verifiable Presentations and Verifiable Credentials in any Credential Format. This section defines a set of Credential Format specific parameters for some of the known Credential Formats. For the Credential Formats that are not mentioned in this specification, other specifications or deployments can define their own set of Credential Format specific parameters.
+OpenID for Verifiable Presentations is Credential Format agnostic, i.e., it is designed to allow applications to request and receive Verifiable Presentations and Verifiable Credentials in any Credential Format. This section defines a set of Credential Format specific parameters and rules for some of the known Credential Formats. For the Credential Formats that are not mentioned in this specification, other specifications or deployments can define their own set of Credential Format specific parameters.
 
 ## W3C Verifiable Credentials
 
-W3C Verifiable Credentials may use an additional parameter for the `descriptor_map` with the `presentation_submission`: The `path_nested` object inside an Input Descriptor Mapping Object is used to describe how to find a returned Credential within a Verifiable Presentation, and contains a `format` parameter with the Credential format identifier as a value and a `path` parameter with a relative path to the Verifiable Credential. Non-normative examples can be found further in this section.
+
+### DCQL
+
+#### Parameters in the `meta` parameter in Credential Query
```

Comment:
One issue i see for w3c VCs is that issuer based matching with DCQl can become quite complex, as you don't know whether to query `['issuer']` or `['issuer', 'id']`. 

So if you would use claim sets, you'd have to duplicate all claim sets. I know the WG decided against adding `did` as a trusted authority mechanism, but it would make issuer matching a lot easier IMO.

Reply:
I've opened https://github.com/openid/OpenID4VP/issues/494 to support did as trusted authorities, which would solve this issue, but it's not specific to this PR and can be worked around using claim sets